### PR TITLE
Add ProductHunt Badge to README

### PR DESCRIPTION
**Pull request by Pl4gue**
## Description
Add ProductHunt Badge to README

## What does your PR belong to?
- [ ] Website
- [ ] Snippets
- [x] General / Things regarding the repository (like CI Integration)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (non-breaking improvement of a snippet)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have checked that the changes are working properly
- [x] I have checked that there isn't any PR doing the same
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: static-parts/README-start.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
 ![Logo](/logo.png)
 
-# 30 seconds of code
+# 30 seconds of code [![ProductHunt](https://img.shields.io/badge/producthunt-vote-orange.svg)](https://www.producthunt.com/posts/30-seconds-of-code)
 [![License](https://img.shields.io/badge/license-CC0--1.0-blue.svg)](https://github.com/Chalarangelo/30-seconds-of-code/blob/master/LICENSE) [![Gitter chat](https://img.shields.io/badge/chat-on%20gitter-4FB999.svg)](https://gitter.im/30-seconds-of-code/Lobby) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](http://makeapullrequest.com) [![Travis Build](https://travis-ci.org/Chalarangelo/30-seconds-of-code.svg?branch=master)](https://travis-ci.org/Chalarangelo/30-seconds-of-code) [![Insight.io](https://img.shields.io/badge/insight.io-Ready-brightgreen.svg)](https://insight.io/github.com/Chalarangelo/30-seconds-of-code/tree/master/?source=0) [![js-semistandard-style](https://img.shields.io/badge/code%20style-semistandard-brightgreen.svg)](https://github.com/Flet/semistandard)
 
+### _Vote for this project on [Producthunt](https://www.producthunt.com/posts/30-seconds-of-code)_
```
Comment (skatcat31) [inline static-parts/README-start.md]: Is there a reason there is an underscore before the V?

Comment (atomiks) [reply static-parts/README-start.md]: It's in italics

Comment (skatcat31) [reply static-parts/README-start.md]: ... MD italics is _? I thought it was *?

Comment (skatcat31) [reply static-parts/README-start.md]: *test*_test_ oh it's both

**Pull request merged**